### PR TITLE
[CLI-62] cli command to list tenants available

### PR DESCRIPTION
**Pull request by bright-poku**
### Description

 add a command in internal/cli/tenants.go that lists in a table the tenants available to the user:

```
auth0 tenants list

+-------------------+
| AVAILABLE TENANTS |
+-------------------+
| bap-tenants       |
| bpoku             |
| test-bpoku        |
+-------------------+
```

### References

https://auth0team.atlassian.net/browse/CLI-62?atlOrigin=eyJpIjoiMTlhZjY0ZTY5NTk3NDFkMjk2NjNlZjAyMDg4M2ZlNTEiLCJwIjoiaiJ9

### Testing

> Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.
>
> Also include details of the environment this PR was developed in (language/platform/browser version).

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [X ] The correct base branch is being used, if not `master`

### CODE REVIEW

File: internal/cli/tenants.go [hunk 1]
```diff
@@ -14,9 +16,41 @@ func tenantsCmd(cli *cli) *cobra.Command {
 	}
 
 	cmd.AddCommand(useTenantCmd(cli))
+	cmd.AddCommand(listTenantCmd(cli))
 	return cmd
 }
 
+func listTenantCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "list",
```
Comment (jfatta) [inline internal/cli/tenants.go:25]: I'd suggest adding the `ls` command alias.

File: internal/cli/tenants.go [hunk 2]
```diff
@@ -14,9 +16,41 @@ func tenantsCmd(cli *cli) *cobra.Command {
...
+		Short: "List your tenants",
+		Long: `auth0 tenants list`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			listTenant, err := cli.listTenants()
+			if err != nil {
+				return fmt.Errorf("Unable to load tenants due to an unexpected error: %w", err)
+			}
+
+			tenNames := make([]string, len(listTenant))
```
Comment (jfatta) [inline internal/cli/tenants.go:34]: can we move the render logic to the `display` package? most (all should) follow this approach. Besides consistency, it is going to apply common logic to display 1 vs several results, json output, etc.

**Review by jfatta — COMMENTED** [review]
looks good.

File: internal/cli/tenants.go (hunk 1 above)
Comment (Widcket) [reply internal/cli/tenants.go:25]: Good idea, we could alias `ls` all the `list` commands.

Comment (bright-poku) [reply internal/cli/tenants.go:25]: do you want to create a seperate ticket for that? `ls` all the `list` commands. I will make the changes for this PR

Comment (Widcket) [reply internal/cli/tenants.go:25]: Yup, I'll create a ticket for that

**Review by jfatta — APPROVED** [review]
🚀

**Pull request merged**